hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java

```diff
@@ -44,6 +44,8 @@ class EthTxDataTest {
             "f864012f83018000947e3a9eaf9bcc39e2ffa38eb30bf7a93feacbc18180827653820277a0f9fbff985d374be4a55f296915002eec11ac96f1ce2df183adf992baa9390b2fa00c1e867cc960d9c74ec2e6a662b7908ec4c8cc9f3091e886bcefbeb2290fb792";
     static final String RAW_TX_TYPE_0_TRIMMED_LAST_BYTES =
             "f864012f83018000947e3a9eaf9bcc39e2ffa38eb30bf7a93feacbc18180827653820277a0f9fbff985d374be4a55f296915002eec11ac96f1ce2df183adf992baa9390b2fa00c1e867cc960d9c74ec2e6a662b7908ec4c8cc9f3091e886bcefbeb2290000";
+    static final String RAW_TX_TYPE_1 =
+            "01f87182012a8085a54f4c3c00832dc6c094000000000000000000000000000000000000052d8502540be40083123456c001a04d83230d6c19076fa42ef92f88d2cb0ae917b58640cc86f221a2e15b1736714fa03a4643759236b06b6abb31713ad694ab3b7ac5760f183c46f448260b08252b58";
```

Comment:
I knew this was coming.  Another couple of tests filled with magical hex strings that came from somewhere else, pasted in here, along with a bunch of finicky code to confirm ... something.  Sigh.  The worst part of dealing with these contract encodings/decodings in Java is that we lack the support throughout the repo for writing this kind of test in a more reader-friendly manner.  Sigh.

Reply:
We should add the transaction object in comments to expose this

Reply:
Just added the tx object as a comment. In the entire PR, I followed the existing file/design patterns, and there is no one tx object as a comment, that's why I missed it. It will be very useful for further debugging/editing.